dfsnew/backupinfo_test.go

```diff
@@ -0,0 +1,107 @@
+// Copyright 2015 The Serviced Authors.
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// +build unit
+
+package dfs_test
+
+import (
+	"archive/tar"
+	"bytes"
+	"encoding/json"
+	"time"
+
+	. "github.com/control-center/serviced/dfsnew"
+	"github.com/control-center/serviced/domain/host"
+	"github.com/control-center/serviced/domain/pool"
+	"github.com/control-center/serviced/domain/servicetemplate"
+	. "gopkg.in/check.v1"
+)
+
+func (s *DFSTestSuite) TestBackupInfo_NotATar(c *C) {
+	buf := bytes.NewBufferString("this is not a tarfile")
+	binfo, err := s.dfs.BackupInfo(buf)
+	c.Assert(binfo, IsNil)
+	c.Assert(err, NotNil)
```

Comment:
I don't know if the error necessarily matters, as there isn't any complicated logic in this particular function